### PR TITLE
New/upload svg to s3 bucket

### PR DESCRIPTION
**Pull request by ovchynnikov**

### CODE REVIEW

**Review by mykytabatrak — APPROVED** [review]

File: backend/package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "@types/github-slugger": "1.3.0",
     "@types/jsonwebtoken": "8.5.4",
     "@types/markdown-it": "12.2.1",
+    "@types/mime-types": "^2.1.1",
```
Comment (aleksandr-kuharenko) [inline backend/package.json]:
```suggestion
    "@types/mime-types": "2.1.1",
```

File: backend/package.json [hunk 2]
```diff
@@ -60,6 +61,7 @@
     "jsonwebtoken": "8.5.1",
     "jwks-rsa": "1.11.0",
     "markdown-it": "12.2.0",
+    "mime-types": "^2.1.32",
```
Comment (aleksandr-kuharenko) [inline backend/package.json]:
```suggestion
    "mime-types": "2.1.32",
```

File: backend/src/common/helpers/s3-file-storage.helper.ts [hunk 3]
```diff
@@ -31,11 +32,15 @@ export const uploadFile = (
...
   const fileStream = fs.createReadStream(file.path);
+
+  const type = mime.contentType(file.path) as string;
+
   const uploadParams = {
     Bucket: bucketName,
     Body: fileStream,
     Key: file.filename,
     ACL: 'public-read',
+    ContentType: type,
```
Comment (aleksandr-kuharenko) [inline backend/src/common/helpers/s3-file-storage.helper.ts]:
```suggestion
  const type = mime.contentType(file.path);

  const uploadParams = {
    Bucket: bucketName,
    Body: fileStream,
    Key: file.filename,
    ACL: 'public-read',
    ContentType: type || undefined,
```

**Comment by aleksandr-kuharenko** [comment]
Probably we should allow svg files here: https://github.com/BinaryStudioAcademy/bsa-2021-infostack/blob/develop/backend/src/common/helpers/multer.helper.ts#L13

**Comment by ovchynnikov** [comment]
> Probably we should allow svg files here: https://github.com/BinaryStudioAcademy/bsa-2021-infostack/blob/develop/backend/src/common/helpers/multer.helper.ts#L13

yes, I've done it before and it is mandatory but suddenly started doing email bug fixing and just messed on with this commit.

**Review by aleksandr-kuharenko — APPROVED** [review]

**Pull request merged**